src/Vendr.PaymentProviders.Worldpay/WorldpaySettings.cs

```diff
@@ -1,10 +1,58 @@
 using Vendr.Core.Web.PaymentProviders;
 
-namespace Vendr.PaymentProviders
+namespace Vendr.PaymentProviders.Worldpay
 {
     public class WorldpaySettings
     {
-        [PaymentProviderSetting(Name = "Continue URL", Description = "The URL to continue to after this provider has done processing. eg: /continue/")]
+        [PaymentProviderSetting(Name = "Cancel Url", Description = "The Cancel URL")]
+        public string CancelUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Error Url", Description = "The Error URL")]
+        public string ErrorUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Continue Url", Description = "The Continue URL")]
         public string ContinueUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Install ID", Description = "The installation ID")]
+        public string InstallId { get; set; }
+
+        [PaymentProviderSetting(Name = "Mode", Description = "TEST or LIVE")]
+        public string Mode { get; set; }
+
+        [PaymentProviderSetting(Name = "Test Mode Number", Description = "100 for TEST, 0 for LIVE")]
+        public string TestModeNumber { get; set; }
+
+        [PaymentProviderSetting(Name = "Test URL", Description = "Default: https://secure-test.worldpay.com/wcc/purchase")]
+        public string TestUrl { get; set; }
+
+        [PaymentProviderSetting(Name = "Live URL", Description = "Default: https://secure.worldpay.com/wcc/purchase")]
+        public string LiveUrl { get; set; }
+        
+        [PaymentProviderSetting(Name = "Auth Mode", Description = "A for a full authorisation, or E for a pre-authorisation")]
+        public string AuthMode { get; set; }
```

Comment:
Maybe consider using an Enum here as Vendr will convert that to a dropdown. You can use Description attributes on the enum values to give friendly display names.

````csharp
public enum WorldpayAuthMode
{
    [Description("Full Authorization")]
    A,
    [Description("Pre-Authorization")]
    E
}
````